crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -1,145 +1,40 @@
-use crate::Anchor;
-use bevy_asset::Handle;
+use crate::TextureAtlasLayout;
+use bevy_asset::{Assets, Handle};
 use bevy_ecs::component::Component;
-use bevy_math::{Rect, Vec2};
-use bevy_reflect::{FromReflect, Reflect, TypeUuid};
-use bevy_render::{color::Color, texture::Image};
-use bevy_utils::HashMap;
-
-/// An atlas containing multiple textures (like a spritesheet or a tilemap).
-/// [Example usage animating sprite.](https://github.com/bevyengine/bevy/blob/latest/examples/2d/sprite_sheet.rs)
-/// [Example usage loading sprite sheet.](https://github.com/bevyengine/bevy/blob/latest/examples/2d/texture_atlas.rs)
-#[derive(Reflect, FromReflect, Debug, Clone, TypeUuid)]
-#[uuid = "7233c597-ccfa-411f-bd59-9af349432ada"]
-#[reflect(Debug)]
+use bevy_math::Rect;
+use bevy_reflect::Reflect;
+
+/// Component used to draw a specific section of a texture.
+///
+/// It stores a handle to [`TextureAtlasLayout`] and the index of the current section of the atlas.
+/// The texture atlas contains various *sections* or *cuts* of a given texture, allowing users to have a single
```

Comment:
```suggestion
/// The texture atlas contains various *sections* of a given texture, allowing users to have a single
```

There's no need to confuse users with synonymous definitions.